src/Compilers/Core/CodeAnalysisTest/InternalUtilities/OneOrManyTests.cs

```diff
@@ -265,7 +266,7 @@ public void SequenceEqual()
         [Fact]
         public void SequenceEqual_WithComparer()
         {
-            var comparer = new TestEqualityComparer<int>((x, y) => x % 10 == y % 10);
+            var comparer = EqualityComparer<int>.Create((x, y) => x % 10 == y % 10);
```

Comment:
>EqualityComparer<int>.Create((x, y) => x % 10 == y % 10); [](http://example.com/codeflow?start=27&length=57)

This replacement doesn't look equivalent.

Reply:
Because GetHashCode used to not throw NSE? The test does not use the comparer for hashing so they are equivalent for the purpose of the test. All tests pass.